tox.ini

```diff
@@ -84,9 +84,10 @@ commands =
 description = Run integration tests
 deps =
     juju
+    pylxd
     pytest
     pytest-asyncio
-    pytest-operator
+    pytest-operator @ git+https://github.com/charmed-kubernetes/pytest-operator@akd/add-k8s-alpha
```

Comment:
how are we feeling about this now?

Reply:
It's awesome, it simplifies the add-k8s logic a lot!

Reply:
So we can merge with this in here.  I'll make a card to release this